core/common/src/main/java/alluxio/security/authorization/Permission.java

```diff
@@ -205,4 +205,27 @@ public int hashCode() {
   public String toString() {
     return mOwner + ":" + mGroup + ":" + mMode;
   }
+
+  /**
+   * Gets the owner bits from the mode in short format. E.g. mode 0775 returns 0007; mode 0500
+   * returns 0005.
+   *
+   * @param mode the mode in short format
+   * @return the owner bits in short format
+   */
+  public static short getOwnerBits(short mode) {
```

Comment:
would it make more sense to have this be a non-static method that operates on a permission?

`permission.getOwnerBits()` and similarly `permission.setOtherBits()`

we should also add all the combinations [get|set][Owner|Group|Others]Bits

the argument to the set method should be of type `Mode.Bits` and the implementation of these methods should be relying on the `Bits` methods ... in other words, the only class that deals with bit arithmetic should be `Bits`